MaxText/max_utils.py

```diff
@@ -351,3 +348,31 @@ def _cross_entropy_with_logits_bwd(
 
 cross_entropy_with_logits.defvjp(_cross_entropy_with_logits_fwd,
                                  _cross_entropy_with_logits_bwd)
+
+
+def get_train_shardings(mesh, state_mesh_annotations, config):
```

Comment:
Surely this is a maxtext_util? MaxText utils are MaxText specific..

Reply:
Agreed, done.